COLLABORATOR_GUIDE.md

```diff
@@ -0,0 +1,201 @@
+unclear, the issue should be left open for several days to allow for
+additional discussion. Where this does not yield input from io.js
+Collaborators or additional evidence that the issue has relevance, the
+issue may be closed. Remember that issues can always be re-opened if
+necessary.
+
+## Accepting Modifications
+
+All modifications to the the io.js code and documentation should be
+performed via GitHub pull requests, including modifications by
+Collaborators and TC members.
+
+All pull requests must be reviewed and accepted by a Collaborator with
+sufficient expertise who is able to take full responsibility for the
+change. In the case of pull requests proposed by an existing
+Collaborator, an additional Collaborator is required for sign-off.
+
+In some cases, it may be necessary to summon a qualified Collaborator
+to a pull request for review by @-mention.
+
+If you are unsure about the modification and are not prepared to take
+full responsibility for the change, defer to another Collaborator.
+
+Before landing pull requests, sufficient time should be left for input
+from other Collaborators. Leave at least 48 hours during the week and
+72 hours over weekends to account for international time differences
+and work schedules. Trivial changes (e.g. those which fix minor bugs
+or improve performance without affecting API or causing other
+wide-reaching impact) may be landed after a shorter delay.
+
+Where there is no disagreement amongst Collaborators, a pull request
+may be landed given appropriate review. Where there is discussion
+amongst Collaborators, consensus should be sought if possible. The
+lack of consensus may indicate the need to elevate discussion to the
+TC for resolution (see below).
+
+All bugfixes require a test case which demonstrates the defect. The
+test should *fail* before the change, and *pass* after the change.
+
+### Involving the TC
+
+Collaborators may opt to elevate pull requests or issues to the TC for
+discussion by assigning the ***tc-agenda*** tag. This should be done
+where a pull request:
+
+- has a significant impact on the codebase,
+- is inherently controversial; or
+- has failed to reach consensus amongst the Collaborators who are
+  actively participating in the discussion.
+
+The TC should serve as the final arbiter where required.
+
+## Landing Pull Requests
+
+Always modify the original commit message to include additional meta
+information regarding the change process:
+
+- A `Reviewed-By: Name <email>` line for yourself and any
+  other Collaborators who have reviewed the change.
+- A `PR-URL:` line that references the full GitHub URL of the original
+  pull request being merged so it's easy to trace a commit back to the
+  conversation that lead up to that change.
+- A `Fixes: X` line, where _X_ is either includes the full GitHub URL
+  for an issue, and/or the hash and commit message if the commit fixes
+  a bug in a previous commit.
```

Comment:
As a final minor nit, it might be useful to link to a commit or two in the repo as examples of what's expected.